test/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM python:3.5.2
+FROM hashicorp/terraform as terraform
 
-ENV TERRAFORM_VERSION 0.11.8
+FROM python:3.6
+
+ENV TERRAFORM_VERSION 0.12.16
```

Comment:
Is there any need for this line? If not please remove it.

Reply:
I'm struggling to see which line you mean exactly. Is it `TERRAFORM_VERSION`? I'm guessing since we're using the `hashicorp/terraform` image that this ends up being redundant...

I'll remove it.

Reply:
Yes, the TERRAFORM_VERSION.